File: table.py
from datetime import datetime
from sqlalchemy import Column, ForeignKey, Integer, String, DateTime, Float, Boolean
from sqlalchemy.ext.declarative import declarative_base
from sqlalchemy.orm import relationship
from sqlalchemy import Table, create_engine

Base = declarative_base()

class table(Base):
    __tablename__ = 'table'
    ID = Column(Integer, primary_key=True)
    branchID = Column(Integer, nullable=False, ForeignKey('branch.ID'))
    numOfSeats = Column(Integer)
    reserved = Column(Boolean)

    # relations
    branch = relationship("customer", back_populates="tables")

File: AndroidAPP/agdd 7aga/Phonograph/app/src/main/java/com/example/android/phonograph/Table.java
package com.example.android.phonograph;

public class Table implements Comparable< Table >{
    String id;
    String BranchID;
    String Num_of_Seats;
    Boolean Reserved;
    public Table(String id, String branchID, String num_of_Seats, Boolean reserved) {
        this.id = id;
        BranchID = branchID;
        Num_of_Seats = num_of_Seats;
        Reserved = reserved;
    }

    @Override
    public int compareTo(Table o) {
        return this.Num_of_Seats.compareTo(o.Num_of_Seats);
    }
}
